docs/develop/java/failure-detection.mdx

```diff
@@ -189,7 +189,7 @@ To set a Retry Policy, known as the [Retry Options](/encyclopedia/retry-policies
                 .build();
   ```
 
-### Overriding the retry interval with Next Retry Delay {#next-retry-delay}
+### Overriding the retry interval with Next Retry Delay {#activity-next-retry-delay}
```

Comment:
We have a new policy for SEO that use "Override" rather than Overriding".

You may have to update a link or links that point here: 

```
docs% rgrep -l mdx "detection#next-retry-delay"
./docs/references/failures.mdx
./docs/encyclopedia/retry-policies.mdx
```

Reply:
I think Next Retry Delay may be an existing mistake, with all caps. I think it's "next Retry delay" based on Vale.